### DISCUSSION TITLE
Welcome to Swiss-QR-Code-integration Discussions!

### DISCUSSION THREAD

**Discussion by sradloff** [document-capture/swiss-qr-code#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TraserCBO** [comment]
Hi, as i took a deeper look into the code, i have one question: I think it is possible that the OCR does not recognize a QR Code (maybe bad scan or something else...). In this case i would not have entries in the Document Word Table with Type "QRCode". Do i have any option to capture the QR Code then manually? Or is the recognization of the QR Codes so good that this won't happen?
A customer using this is asking for a feedback before registering Document if QR Code has been read. So i am thinking about giving her a template field where she can capture the code if engine didn't. But i am not sure if this is possible... Kind regards, Christian

**Reply by sradloff** [reply]
Hi,
you cannot put the whole QR into a field due to the length limitations of BC.
So if the scans aren't that worse, I would use the automized process of standard functionality in BC.

**Reply by TraserCBO** [reply]
So there isn't any chance to let the user capture the QR Code manually, if it is not recognized? Or the other way round: If QR Code is not captured by OCR engine, it can't be captured by user either?

**Comment by sha-0212** [comment]
Hi, first I imported in the customer BC Public (20.1) the V1.0.0.0 app. After that I wanted to import the V2.0.0.0 app. On the upload from V2.0.0.0 I got the following error:

Deployment failed. Errors: App ID : f27aa103-a4c5-4886-a6d0-60134c884e26 Message : { Upgrade der NAV-Erweiterung 'PTE DC Swiss QR-Bill Management by github.com/document-capture 2.0.0.0' aufgrund des folgenden Fehlers fehlgeschlagen: 'TableExtension 61110 DCADV QR Code Doc. Cat. Ext. :: Die Tabelle „CDC Document Category“ wurde nicht gefunden. Das Entfernen von Tabellen ist nicht zulässig, es sei denn, sie sind temporär oder werden bei einem Migrationsvorgang in eine andere App verschoben.'. }

Can you please help me or give me an input what I am doing probably wrong?  

Thank you and best regards
Sereina
